cli/flox-core/src/activations.rs

```diff
@@ -367,8 +385,29 @@ pub fn write_activations_json<V: Serialize>(
 
 #[cfg(test)]
 mod test {
+    use std::process::{Child, Command};
+
     use super::*;
 
+    // NOTE: these two functions are copied from flox-rust-sdk since you can't
+    //       share anything behind #[cfg(test)] across crates
```

Comment:
nb
are they in a test_helper module that we could mark with `cfg(feature = "tests")`?

Looks like the bigger problem would be impoirting -sdk into flox-core which would be a circular dependency.

Reply:
This was a copy/pasta from `flox-watchdog`. Turns out they're not even in `flox-rust-sdk` after 1e33841bc21826fdcaf1a26be85f1a7b1606f271, so `flox-watchdog` could take a `test_helper` from `flox-core`.